components/Button/button.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import Button from './button'; 
+
+describe('Button Component', () => {
+  it('renders the button with the "Submit" text', async() => {
+    const { getByText, container } = render(<Button />);
+    await waitFor(() => {
```

Comment:
no need to use waitfor

`render(<Button />);
 expect(screen.getByText("Submit")).toBeInTheDocument();
`